src/components/Attachments/AttachmentCarousel/extractAttachmentsFromReport.ts

```diff
@@ -7,16 +7,15 @@ import CONST from '@src/CONST';
 import type {ReportAction, ReportActions} from '@src/types/onyx';
 
 /** Constructs the initial component state from report actions */
-function extractAttachmentsFromReport(parentReportAction: ReportAction, reportActions: ReportActions) {
-    const actions = [parentReportAction, ...ReportActionsUtils.getSortedReportActions(Object.values(reportActions))];
+function extractAttachmentsFromReport(parentReportAction?: ReportAction, reportActions?: ReportActions) {
+    const actions = [parentReportAction, ...ReportActionsUtils.getSortedReportActions(Object.values(reportActions ?? {}))];
```

Comment:
```suggestion
    const actions = [...(parentReportAction ? [parentReportAction] : []), ...ReportActionsUtils.getSortedReportActions(Object.values(reportActions ?? {}))];
```
actions should have type ReportAction[]